app/src/main/res/layout/revision_card_activity.xml

```diff
@@ -22,7 +22,7 @@
         android:id="@+id/revision_card_toolbar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/component_color_revision_card_activity_toolbar_color"
+        android:background="@color/component_color_concept_card_fragment_toolbar_color"
```

Comment:
We should change component_color_revision_card_activity_toolbar_color instead of using concept card's here (since we're effectively changing the revision title color in this PR).

Reply:
done